docs/sharing/publish-storybook.md

```diff
@@ -129,17 +129,32 @@ When you publish Storybook, you also get component history and versioning down t
 
 ![Library history in Chromatic](./workflow-history-versioning.png)
 
-## Publish Storybook to other services
+ ## Publish Storybook to other services
```

Comment:
@mickmcgrath13 just a bit of a nitpick, but it seems that there was an extra space added here. Can you remove it?

Reply:
fixed with latest commit: https://github.com/storybookjs/storybook/pull/24527/commits/abb7942c5f3b910521d107d1680ea46007431d2d